exercises/K-functions-parameters/exercise.js

```diff
@@ -1,6 +1,7 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+function multiply(num1, num2) {
   // Calculate the result of the function and return it
+  return num1 / num2;
```

Comment:
**Incorrect operator**

Needs a multiplication operator - i.e. not a division operator